app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -179,6 +179,7 @@ public void onPageSelected(int position) {
                         Timber.d("Contributions tab selected");
                         tabLayout.getTabAt(CONTRIBUTIONS_TAB_POSITION).select();
                         isContributionsFragmentVisible = true;
+                        ((NearbyParentFragment)contributionsActivityPagerAdapter.getItem(NEARBY_TAB_POSITION)).hideKeyboard();
```

Comment:
Why not make a function in utils & call it from the activity and not involve the NearbyFragment?

Reply:
Sir, I have made the changes as suggested. There was a Utils function already defined which could be reused for this implementation. @ashishkumar468